CMakeLists.txt

```diff
@@ -57,15 +56,38 @@ if (${LLVM_PACKAGE_VERSION} VERSION_LESS "5.0")
   message(FATAL_ERROR "LLVM version must be >= 5.0")
 endif()
 
+if (NOT "${CMAKE_SYSTEM_NAME}" STREQUAL "Windows")
+  if (${LLVM_PACKAGE_VERSION} VERSION_LESS "10.0")
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
+  else()
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++14")
```

Comment:
I think we can just bump the minimum requirement to C++14 at this point to keep things simple. We should also do this using CMake's builtin capability to make this is a bit more flexibile. Can you try this please?
```
# Require C++14
set(CMAKE_CXX_STANDARD 14)
set(CMAKE_CXX_STANDARD_REQUIRED ON)
```

Reply:
Fixed in latest version.